src/Services/Qdm/QdmRequestSome.php

```diff
@@ -28,39 +24,18 @@ class QdmRequestSome implements QdmRequestInterface
      */
     public function __construct(array $pids)
     {
-        $this->pids = $pids;
-
-        if (is_array($pids)) {
-            $this->pidString = implode(",", $pids);
-        }
-
+        $filterClause = str_replace("?,", count($pids));
```

Comment:
@kchapple Doesn't make sense. Missing subject in str_replace.

Reply:
sorry @sjpadgett is supposed to be str_repeat(), auto-complete beat me this time.

Reply:
That worked so thank you sir. I'll bring in once you update PR.